packages/legend-query-builder/src/components/__tests__/QueryBuilderRunQuery.test.tsx

```diff
@@ -71,8 +72,10 @@ test(
     const param = renderResult.getByRole('dialog');
     const graphManager = queryBuilderState.graphManagerState.graphManager;
     const pureManager = guaranteeType(graphManager, V1_PureGraphManager);
+    const executionResultMap = new Map<string, string>();
+    executionResultMap.set(V1_EXECUTION_RESULT, mocked);
     createSpy(pureManager.engine, 'runQueryAndReturnString').mockResolvedValue(
-      mocked,
+      executionResultMap,
```

Comment:
Why not just do the same thing here as in your other tests?
```suggestion
      { executionResult: mocked, }
```

Reply:
This function returns a Map not a composite object, therefore I created a Map first